src/content/docs/using-new-relic/welcome-new-relic/get-started/glossary.mdx

```diff
@@ -26,7 +26,9 @@ A glossary of common terminology you may encounter.
     id="account-dropdown"
     title="account dropdown"
   >
-    When you're logged into New Relic, the account dropdown menu displays your login name and is located at the top right of the UI. Clicking this gives you access to various [account-related abilities](/docs/accounts).
+    When you're logged into New Relic, the account dropdown menu displays your login name and is located at the top left of the UI. To switch accounts, click the account switcher dropdown in the top left. 
+    
+    To access your account settings, click your profile icon in the top right. Clicking this gives you access to various [account-related abilities](/docs/accounts).
```

Comment:
Thoughts on including a simple screenshot that shows which one is which?

Reply:
It could work. I'll see what I can do. No other glossary entry has a screenshot, as far as I can tell.

Reply:
I added an image and really took the hedge clippers to the text in this entry. Let me know what you think!